app.py

```diff
@@ -147,14 +154,14 @@ def run_all_tasks():
     list_files()
     delete_file()
     create_user(
-        name + '@test.com',
-        name + '@123',
-        name
+        id + '@test.com',
+        id + '@123',
+        id
     )
     list_user()
 
 
 if __name__ == "__main__":
-
+    warnings.filterwarnings("ignore")
```

Comment:
Not a Python expert, what is this line used for?

Reply:
It ignore all the unsecure https request warnings.

Reply:
@eldadfux It doesn't print any `warnings` in the terminal.

Reply:
@sparkingdark @Nibba2018 wouldn't it be better to throw warning when missing API key or project ID? Don't think that having no warnings is the best solution for this.

Reply:
@eldadfux Yes you are right. Actually #29 has implemented what we need i.e only the insecure warning is ignored and the rest of the warnings are allowed.